ctru-sys/Cargo.toml

```diff
@@ -12,19 +12,42 @@ license = "Zlib"
 links = "ctru"
 edition = "2021"
 
+[features]
+default = []
+
+## Enables generating C++/Rust layout comparison tests.
+## Downstream users of `ctru-sys` shouldn't need to use this feature.
+layout-tests = [
+    "dep:cpp_build",
+    "dep:proc-macro2",
+    "dep:quote",
+    "dep:regex",
+    "dep:rust-format",
+]
+
+[[test]]
+name = "layout_test"
+required-features = ["layout-tests"]
+
 [dependencies]
-libc = { version = "0.2.121", default-features = false }
+libc = { workspace = true }
 
 [build-dependencies]
-bindgen = { version = "0.65.1", features = ["experimental"] }
+bindgen = { version = "0.69", features = ["experimental"] }
 cc = "1.0"
+cpp_build = { optional = true, git = "https://github.com/mystor/rust-cpp.git" }
```

Comment:
Mostly so I could use https://github.com/mystor/rust-cpp/pull/111 which hasn't been released yet. We could do without it and just re-define all the `cc` configuration, or pin the commit hash, but I wasn't too worried about it since this is a test-only dependency